app/controllers/navigation/Navigator.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.navigation
+
+import config.AppConfig
+import javax.inject.Inject
+import models.requests.JourneyRequest
+import models.responses.FlashKeys
+import play.api.mvc.{Result, Results}
+
+class Navigator @Inject()(appConfig: AppConfig) {
```

Comment:
Can we do this with provider and `play.api.Configuraiton` ?

Reply:
I cant store it as a reusable variable that way. The Time to Live needs to be used in multiple places

Reply:
It always could be bind and injected to multiple places

Reply:
I think we need to agree we as a team if we do/don't want to go with that approach in the future. I personally have never found `AppConfig` causes any issues